webapp/src/main/java/com/box/l10n/mojito/entity/RepositoryLocaleAIPrompt.java

```diff
@@ -0,0 +1,66 @@
+package com.box.l10n.mojito.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Index;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(
+    name = "repository_locale_ai_prompt",
+    indexes = {
+      @Index(
+          name = "I__REPOSITORY_LOCALE_AI_PROMPT__REPO_ID__AI_PROMPT_ID",
+          columnList = "repository_id, locale_id, ai_prompt_id")
+    })
+public class RepositoryLocaleAIPrompt extends BaseEntity {
+
+  @ManyToOne
+  @JoinColumn(name = "repository_id")
+  private Repository repository;
+
+  @ManyToOne
+  @JoinColumn(name = "locale_id")
```

Comment:
Should we explicitly mark this column as `nullable = true` to account for our special use of it to mean non-locale specific? 

Also - should the repository column be marked as `nullable = false` - or do we plan to also support something similar there? (global non-repo specific prompts?)

Reply:
No plans for global non repo specific prompts, currently for a repository to be considered enabled for AI translation it needs to have an entry in the repository_locale_ai_prompt table. Prompts themselves can be re-used across repositories, but the individual repos will need an entry here to ensure they are AI translated